app/assets/javascripts/components/contact.es6.jsx

```diff
@@ -0,0 +1,21 @@
+class Contact extends React.Component {
+  render () {
+    return (
+      <div className="contact page-wrapper">
+        <Header />
+          <div className="content-wrapper">
+            <form className="contact-form" action="demo_form.asp">
+              <p className="questions">Any questions, concerns or suggestions?  We're all ears.</p>
+              <input type="text" className="input-field half-width-input" name="first-name" placeholder="First Name"></input>
+              <input type="text" className="input-field half-width-input last-name-input" name="last-name" placeholder="Last Name"></input>
+              <input type="text" className="input-field full-width-input" name="email" placeholder="Email Address"></input>
+              <input type="text" className="input-field full-width-input subject-input" name="subject" placeholder="Subject"></input>
+              <textarea type="text" className="input-field full-width-input message-input" name="message" placeholder="Message"></textarea>
+              <Button className="button--orange">Submit</Button>
```

Comment:
for Buttons, just use <button> not hte Bootstrap one. Pull from master, I made a few styling changes to the button. Also, for classname, it should be "button button--orange" always prefix it with button and add extra classnames